charmcraft/reactive_plugin.py

```diff
@@ -140,6 +141,26 @@ def get_build_commands(self) -> List[str]:
         return [" ".join(shlex.quote(i) for i in command)]
 
 
+def run_charm_tool(args: List[str]):
+    """Run the charm tool, log and check exit code."""
+    result_classification = "SUCCESS"
+    exc = None
+
+    try:
+        completed_process = subprocess.run(args, check=True)
+    except subprocess.CalledProcessError as call_error:
+        exc = call_error
+        if call_error.returncode < 100 or call_error.returncode >= 200:
+            result_classification = "ERROR"
+            raise
+        else:
```

Comment:
this "else" is not needed, as the "block of True" of the if will not allow these lines to continue.